crypto/keys/codec.go

```diff
@@ -0,0 +1,25 @@
+package keys
+
+import (
+	cryptoAmino "github.com/tendermint/tendermint/crypto/encoding/amino"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/crypto/keys/hd"
+	emintCrypto "github.com/cosmos/ethermint/crypto"
+	cosmosKeys "github.com/cosmos/cosmos-sdk/crypto/keys"
+)
+
+var cdc *codec.Codec
+
+func init() {
+	cdc = codec.New()
+	cryptoAmino.RegisterAmino(cdc)
+	cdc.RegisterInterface((*cosmosKeys.Info)(nil), nil)
+	emintCrypto.RegisterCodec(cdc)
+	cdc.RegisterConcrete(hd.BIP44Params{}, "crypto/keys/hd/BIP44Params", nil)
+	cdc.RegisterConcrete(localInfo{}, "crypto/keys/localInfo", nil)
+	cdc.RegisterConcrete(ledgerInfo{}, "crypto/keys/ledgerInfo", nil)
+	cdc.RegisterConcrete(offlineInfo{}, "crypto/keys/offlineInfo", nil)
+	// cdc.RegisterConcrete(multiInfo{}, "crypto/keys/multiInfo", nil)
```

Comment:
why is it commented out?

Reply:
To allow for it to be easily added in the future if it is determined that the functionality is desired? I can remove all references now, I just assumed since it was a more opinionated decision to keep or to add later I would just leave these references commented out for now.

Reply:
Can we maybe add a comment to explain that ?